rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/asset_search/action.py

```diff
@@ -0,0 +1,34 @@
+import insightconnect_plugin_runtime
+from .schema import AssetSearchInput, AssetSearchOutput, Input, Output, Component
+
+
+# Custom imports below
+from ..start_scan.action import format_body
+
+
+class AssetSearch(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="asset_search", description=Component.DESCRIPTION, input=AssetSearchInput(), output=AssetSearchOutput()
+        )
+
+    def run(self, params={}):
+        asset_crit = params.get(Input.ASSET_CRITERIA)
+        vuln_crit = params.get(Input.VULN_CRITERIA)
+        size = params.get(Input.SIZE, 0)
```

Comment:
Just realized, should we default size to say 200? Along with default that comes from plugin spec (change that too)